.vitest/vitest.shared.ts

```diff
@@ -3,6 +3,11 @@ import { configDefaults, defineConfig } from "vitest/config";
 
 export const sharedConfig = defineConfig({
   test: {
+    typecheck: {
+      enabled: process.env.TYPECHECK === "true",
```

Comment:
So using a ENV because I should have been able to pass this as an argument to the vitest. 
After hours of it not working, I did this env and it just works. So 🤷‍♀️